Sources/SwiftProtobuf/Map.swift

```diff
@@ -17,7 +17,7 @@
 import Foundation
 import Swift
 
-public struct ProtobufMap<KeyType: MapKeyType, ValueType: MapValueType>
+public struct Map<KeyType: MapKeyType, ValueType: MapValueType>
```

Comment:
I think leaving `Proto` on these types would be fine, since as you say, they're metadata related specifically to the protobuf nature of these types. They're also unlikely to be used by clients unless they're doing something advanced like writing their own visitor; most users aren't going to see them, so the slightly more "technical" name doesn't really uglify code.